File: Day4ProgrammingFoundation/ModelPython/Customer.py
class Customer_model:
    def __init__(self, name, age, salary):
        self.name = name
        self.age = age
        self.salary = salary


class Risk_Assesor:
    def assess(customer):
        if customer.age < 21:
            return "Not Eligible"
        elif customer.salary >= 8000000:
            return "Eligible"
        elif customer.salary >= 3000000:
            return "Manual Review"
        else:
            return "Not Eligible"

File: Day3ProgrammingFoundation/Customer.java
import java.util.jar.Manifest;

public class Customer{
    String name;
    int age;
    double salary;

    public Customer (String name, int age, double salary){
        this.name = name;
        this.age = age;
        this.salary = salary;
    }

    

    public static void main (String[] args){
        Customer customer1 = new Customer("Rina", 24, 4_000_000);
        System.out.println(customer1.name);
        System.out.println(customer1.age);
        System.out.println(customer1.salary);
    }
}